canonical/ubuntu.com
# Display Card form when there is no saved card

## Done
Display Card form when there is no saved card 

The view when there's no saved card

- Before fix 
![image](https://user-images.githubusercontent.com/57550290/201980036-d1faee89-5575-41c8-ad0a-f95696efd5d0.png)

- After fix

![image](https://user-images.githubusercontent.com/57550290/201980731-26a818c1-ad9b-4ba3-83cb-4fc55019891e.png)

The view when there's saved card  
<img width="433" alt="image" src="https://user-images.githubusercontent.com/57550290/201978233-55d8c518-8b16-49be-83e4-8e5f93ea13dd.png">

## QA

- Create an UO account and login
- Go to /pro/subscribe
- Check a form is displaying when there's no card saved in the payment method field.
- Also check saved card info is displaying when saved card exists. 

## Issue / Card

Fixes #https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?modal=detail&selectedIssue=WD-394
